x-pack/plugins/cloud_security_posture/common/scripts/get_tests.js

```diff
@@ -0,0 +1,367 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * Dedicated app for output visualization: https://codesandbox.io/p/sandbox/zen-smoke-vxgs2c
+ * Just copy the generated content of 'cso_test_log.json' into the 'data.json' file in the dedicated app
+ * */
+
+const fs = require('fs');
+const path = require('path');
+const readline = require('readline');
+
+// ==============================
+//           Setup
+// ==============================
+
+// Directories to iterate over
+const FTR_SERVERLESS =
```

Comment:
Some other dirs we might want to cover:
- x-pack/test_serverless/api_integration/test_suites/security/cloud_security_posture/index.ts

also we own `cloud_defend`, `session_view` and `kubernetes_security`

Overall idea for automation: take dirs marked with `@elastic/kibana-cloud-security-posture` in `.github/CODEOWNERS`

Reply:
there is also x-pack/performance/journeys/cloud_security_dashboard.ts but I'm not sure if we care

Reply:
alright, i'll make a separate PR to add more dirs